app/models/template.rb

```diff
@@ -33,12 +33,12 @@ def self.dmptemplate_ids(org)
 
   # Retrieves the most recent version of the template for the specified Org and dmptemplate_id 
   def self.current(org, dmptemplate_id)
```

Comment:
I don't see the need for passing org parameter into this method. At the end, a template belongs to one and only one org. Maybe I missing something.